tests/integration/suite/test_app.py

```diff
@@ -591,6 +591,62 @@ def _app_fail():
     assert e.value.error.status == 422
 
 
+def test_app_has_helmversion(admin_pc, admin_mc, remove_resource):
+    """Test that app is using specified helm version"""
+    app_client = admin_pc.client
+    catalog_client = admin_mc.client
+    catalog_name1 = random_str()
+    catalog_name2 = random_str()
+    app_name1 = random_str()
+    app_name2 = random_str()
+    url = "https://github.com/rancher/integration-test-charts"
+    catalog1 = catalog_client.create_catalog(name=catalog_name1,
+                                             branch="master",
+                                             url=url,
+                                             )
+    catalog2 = catalog_client.create_catalog(name=catalog_name2,
+                                             branch="master",
+                                             url=url,
+                                             helmVersion="helm_v3"
+                                             )
+    remove_resource(catalog1)
```

Comment:
Keep the `remove_resource` right under the create call in case the next create call explodes we will still clean up the first resource.